spec/services/invoices/payments/gocardless_service_spec.rb

```diff
@@ -28,6 +30,14 @@
 
       allow(GoCardlessPro::Client).to receive(:new)
         .and_return(gocardless_client)
+      allow(gocardless_client).to receive(:mandates)
+        .and_return(gocardless_mandates_service)
+      allow(gocardless_mandates_service).to receive(:list)
+        .and_return(gocardless_list_response)
+      allow(gocardless_list_response).to receive(:records)
+        .and_return([GoCardlessPro::Resources::Mandate.new(
```

Comment:
Layout/FirstArrayElementLineBreak: Add a line break before the first element of a multi-line array.